core/src/Kokkos_CudaSpace.hpp

```diff
@@ -57,6 +57,23 @@
 
 #include <Cuda/Kokkos_Cuda_abort.hpp>
 
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UM_TO_HOST
+#ifndef KOKKOS_IMPL_DEFINED_CUDA_PIN_UM_TO_HOST_V
```

Comment:
By the way I do not understand why you need this macro.

Reply:
That's from an old version of the code, it doesn't exist any more

Reply:
I thought that was because I was including the header idirectly in the file where I also instantiate the variable or so, but I guess Davids updates should make this obsolete anyway.